.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+Fixes #. A short summary which serves as a squashed-commit message.
+
+A full description of what's been implemented.
+
+Important changes:
```

Comment:
Also not sure about this. Maybe it should somehow connect to "Acceptance Criteria"?

Reply:
I've been thinking the same, btw. But currently the reviewer should go the issue and check it's stated properly anyway and check the acceptance criteria there. Or you think it's about time to revoke that requirement and fill everything for CR in the PR?

Reply:
IMHO linking is enough. Issue tells what's wanted, PR tells what's done. And PR has to match the issue.